crates/trippy-tui/src/frontend/columns.rs

```diff
@@ -270,9 +269,9 @@ impl Display for ColumnType {
 impl ColumnType {
     /// The name of the column in the current locale.
     #[allow(clippy::cognitive_complexity)]
-    pub(self) fn name(&self) -> Cow<'_, str> {
+    pub(self) fn name(self) -> String {
```

Comment:
@nc7s note the change in signature here, since the macro now already returns a `String`.